### DISCUSSION TITLE
To fix all errors needs to run vendor/bin/phpcbf many times

### DISCUSSION THREAD

**Discussion by vol4onok** [PHP-CS-Fixer/PHP-CS-Fixer#7420] [Q&A]
I see one problem. When I run `phpcbf` it doesn't fix all fixible errors. Why is it happening? Is it possible to fix it somehow?
For example I have the project
When I run phpcbf fist time - in the result I have some fixed errors, but in code I still have fixible errors
When I run second time it fix other errors.

In one project state I run 3 time fixer to fix all errors on the project.

**Comment by Wirone** [comment]
Hi @vol4onok! `phpcbf` is a file provided by [`PHPCodeSniffer`](https://github.com/squizlabs/PHP_CodeSniffer), not PHPCSFixer 😉. Please ask there, since we cannot provide the answer. PS. Our Fixer also sometimes has similar issues, these are under [`topic/priority`](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Atopic%2Fpriority) label.
